### PR TITLE
Updating compute.json for force delete

### PR DESCRIPTION
**Pull request by Aishwarya-C-S**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by Sandido** [comment]
Hi @Aishwarya-C-S , 
Please test this change by generating the SDK files from it, running all tests in Playback mode to ensure nothing is broken, and create at least 1 test to verify this Swagger change behaves as expected. 
Some instructions for this can be found here: [https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/29378/Generating-SDK-code-from-a-local-Swagger](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/29378/Generating-SDK-code-from-a-local-Swagger )

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json [hunk 1]
```diff
@@ -2799,6 +2799,13 @@
             "type": "string",
             "description": "The name of the virtual machine."
           },
+          {
+            "name": "forceDeletion",
```
Comment (pilor) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:2803]: What is the difference in behavior?  The description should make it clear.  A while back there was a thread about enabling optional force delete on VMs when a customer was deleting a resource group.  Is this change related to that?  Was that approach ever finalized?

**Review by pilor — CHANGES_REQUESTED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json (hunk 1 above)
Comment (Aishwarya-C-S) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:2803]: sent you a mail..

**Review by pilor — APPROVED** [review]

**Comment by Sandido** [comment]
@pilor , @Aishwarya-C-S , are you able to remove the Python Braking Changes labels? They are no longer accurate since the Python SDK Track 2 version GA-ed last week.

**Review by Sandido — APPROVED** [review]

**Review by fengzhou-msft — APPROVED** [review]

**Comment by kairu-ms** [comment]
Ruby SDK supports now
https://rubygems.org/gems/azure_mgmt_compute/versions/0.20.0
